Graph.py

```diff
@@ -115,9 +125,10 @@ def main():
                     labels2d = Variable(labels2d)
                     labels3d = Variable(labels3d)
 
-                    inputs = inputs.float()
-                    labels2d = labels2d.float()
-                    labels3d = labels3d.float()
```

Comment:
These 3 variables will now be Doubles instead of Floats when cuda is disabled.   There are a number of things wrong here (recasting variables, no type checking until downstream, bad naming) but this code belongs in an `else:`  below line 131